### DISCUSSION TITLE
A brief and complete description of the functions

### DISCUSSION THREAD

**Discussion by kfilippenok** [pasdoc/pasdoc#208] [Q&A] [answered]
I want only "Desc" in the "Functions and Procedures" (funcsprocs_desc image) section, and everything else in the module section (unit_desc image). I am currently building documentation using "implementation-comments=join". How do I get the desired behavior?

funcsprocs_desc
![funcsprocs_desc](https://github.com/user-attachments/assets/50a1d1ca-e537-47e0-80f0-acabdf8c2ff8)

unit_desc
![unit_desc](https://github.com/user-attachments/assets/f3ccd5f0-727a-41dd-8047-373244158d99)

**Comment by kfilippenok** [comment]
In **Interface**
`// Desc.`
`function IsValidInvNum(const InvNum: String): Boolean;`

In **Implementation** 
`// Desc. Just more symbols aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa`
`// aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa`
`// aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa`
`// aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa`
`// aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa.`
`function IsValidInvNum(const InvNum: String): Boolean;`

**Comment by kfilippenok** [comment, marked as answer]
**Solution**: Add [--auto-abstract](https://pasdoc.github.io/AutoAbstractOption)

**Comment by michaliskambi** [comment]
Sorry for a delay in answering! Busy week.

Yes, using abstract (whether automatic by https://pasdoc.github.io/AutoAbstractOption or explicit by `@abstract`  https://pasdoc.github.io/AbstractTag ) is some solution to this.
